ydb/docs/en/core/troubleshooting/examples/schemas/overloaded-shard-1.md

```diff
@@ -0,0 +1,306 @@
+# Overloaded shard example
+
+This article describes an example of how to diagnose overloaded shards and resolve the issue.
+
+For more information about overloaded shards and the causes for this issue, see the article [{#T}](../../performance/schemas/overloaded-shards.md).
+
+The article starts with [stating the problem](#initial-problem). Then we'll take a look at diagrams in Grafana and information on the **Diagnostics** tab in the [Embedded UI](../../../reference/embedded-ui/index.md) to [solve the problem](#solution) and [watch the solution in action](#aftermath).
+
+Also, in the end of the article you can see the steps how to [reproduce the situation](#testbed).
+
+## Initial problem {#initial-problem}
+
+You were notified that your system started taking too long to process user requests.
```

Comment:
Let's explicitly say that these requests access a row-oriented table, which is managed by data shards behind the scenes. With links to the glossary.